layer/compile_time_layer.cc

```diff
@@ -242,7 +242,17 @@ SPL_COMPILE_TIME_LAYER_FUNC(VkResult, CreateShaderModule,
   return res.result;
 }
 
-// Override for vkDestroyDevice.  Removes the dispatch table for the device from
+// Override for vkDestroyShaderModule. Erases the shader module from
+// the layer data.
+SPL_COMPILE_TIME_LAYER_FUNC(void, DestroyShaderModule,
+                            (VkDevice device,
+                             VkShaderModule shader_module,
+                             const VkAllocationCallbacks* allocator)
+                             ) {
```

Comment:
Can you run clang-format on your patch? Either in your editor (format selection) or via CLI:
` git diff main -U0 --no-color | clang-format-diff -p1 -i`